sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -631,6 +631,24 @@ public Read withMaxBufferElementCount(@Nullable Integer maxBufferElementCount) {
           .build();
     }
 
+    /**
+     * Returns a new {@link BigtableIO.Read} that will skip the large rows while reading. This
+     * function will switch the base BigtableIO.Reader class to using the
+     * BigtableReaderWithExperimentalOptions. If
+     *
+     * <p>Does not modify this object.
+     *
+     * <p>This is incompatible with withMaxBufferElementCount()
+     */
+    public Read withExperimentalSkipLargeRows(@Nullable Boolean skipLargeRows) {
```

Comment:
Please dont encode feature status in the public abi. The makes it impossible to evolve the feature into ga status. If this feature is not ready for GA, then please use
[ExperimentalOptions](https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/options/ExperimentalOptions.html) which we already use for BIGTABLE_ENABLE_CLIENT_SIDE_METRICS